### PR TITLE
Expectation / assertion fixes

### PR DESCRIPTION
**Pull request by dnimmo**
A number of the expectations / assertions in the tests here didn't quite match up, and a few of them would pass in situations where they shouldn't as a result.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +76,8 @@
     "Miika Hänninen <miika.hanninen@gmail.com>",
     "Kevin Goslar <kevin.goslar@gmail.com>",
     "John Krull <astrom.flux@gmail.com>",
-    "Maxim Koretskiy <mr.green.tv@gmail.com>"
+    "Maxim Koretskiy <mr.green.tv@gmail.com>",
+    "David Nimmo <dnimmo@gmail.com>"
```
Comment (charlierudolph) [inline package.json]: Please revert the changes to this file. We will bump the version only on new releases and will add all new contributors then.

Comment (dnimmo) [inline package.json]: Ah sorry, I was just following the instructions in the "CONTRIBUTING.md" file :)

**Comment by charlierudolph** [comment]
Can you please give an example of where toBeTruthy / toBeFalsy was not properly testing the method? I'm fine with this change but I don't see anywhere where this test wasn't working properly.

**Comment by dnimmo** [comment]
The tests were all working fine - I was just going through all of the code in order to understand it all, and noticed that there were a number of tests that stated "should be true" but then expected toBeTruthy() etc. - so if the code was changed such that it returned a string/object/etc. it would still pass but the expectation wouldn't have been met. Am I making sense? :D

File: package.json (hunk 1 above)
Comment (charlierudolph) [inline package.json]: The release checklist is just for maintainers. I'll add a note.

**Comment by charlierudolph** [comment]
Searching through the project that are still instance where toBeTruthy / toBeFalsy are used. In your changes you removed some instances of these in favor of toBe(true) / toBe(false) even when the description matched. How do you feel about removing all instance of toBeTruthy / toBeFalsy?

**Comment by dnimmo** [comment]
I removed the ones where the description matched but the setup in the test didn't. I checked every instance of truthy/falsy but some were legitimately being used correctly (there were some where either false or null could be returned for example) so I left those ones as they were. :-)

**Comment by charlierudolph** [comment]
Thanks!
